Dockerfile

```diff
@@ -10,5 +10,7 @@ ENV SRC_DIR=/go/src/github.com/windmilleng/blorgly-backend
 ADD . $SRC_DIR
 
 # Build it:
+RUN go get github.com/gorilla/mux
```

Comment:
usually you would do `cd $SRC_DIR; go get ./...`?

Reply:
i.e., you don't want to hardcode the dependencies in the dockerfile, you want the go tool to read them from your source